### PR TITLE
[RW-7963][RW-7964][risk=low] Add CT access tiers in stable/prod

### PR DESCRIPTION
**Pull request by calbach**
Follow-up PRs will add:

1. Biguery CDRs to both
2. Genomics data to both

### CODE REVIEW

File: api/config/cdr_config_prod.json [hunk 1]
```diff
@@ -7,8 +7,17 @@
...
-      "datasetsBucket": "gs://fc-aou-datasets-registered/",
       "enableUserWorkflows": false
+    },
+    {
+      "accessTierId": 2,
+      "shortName": "controlled",
+      "displayName": "Controlled Tier",
+      "servicePerimeter": "terra_prod_aou_prod_2",
+      "authDomainName": "all-of-us-controlled-prod",
+      "authDomainGroupEmail": "all-of-us-controlled-stable@firecloud.org",
```
Comment (jmthibault79) [inline api/config/cdr_config_prod.json:18]: typo: stable

File: api/config/cdr_config_prod.json [hunk 2]
```diff
@@ -7,8 +7,17 @@
       "servicePerimeter": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_prod",
       "authDomainName": "all-of-us-registered-prod",
       "authDomainGroupEmail": "all-of-us-registered-prod@firecloud.org",
-      "datasetsBucket": "gs://fc-aou-datasets-registered/",
```
Comment (jmthibault79) [inline api/config/cdr_config_prod.json:10]: why remove these?  I don't have much context here.

File: api/libproject/environments.rb [hunk 3]
```diff
@@ -98,6 +98,11 @@ def env_with_defaults(env, config)
         :dest_cdr_project => "fc-aou-cdr-synth-stable",
         :auth_domain_group_email => "GROUP_all-of-us-registered-stable@firecloud.org",
       },
+      "controlled" => {
```
Comment (jmthibault79) [inline api/libproject/environments.rb:101]: missing prod

File: api/config/cdr_config_prod.json (hunk 1 above)
Comment (calbach) [reply api/config/cdr_config_prod.json]: Great catch - that might have been very painful to track down afterwards.

File: api/libproject/environments.rb (hunk 3 above)
Comment (calbach) [reply api/libproject/environments.rb:101]: prod was already here and stable wasn't, not sure why - see L139

File: api/config/cdr_config_prod.json (hunk 2 above)
Comment (calbach) [reply api/config/cdr_config_prod.json:10]: This bucket was never actually used or, perhaps not even created. Currently, we don't have any GCS assets that are part of the CDR in registered tier.

Comment (calbach) [reply api/config/cdr_config_prod.json:10]: As to why it was here: originally the intent was to create a bucket in every perimeter, but we decided not to create extraneous buckets in the end. This was left over from that.

**Pull request merged**